src/istgt_lu_disk.c

```diff
@@ -5113,6 +5116,13 @@ istgt_lu_disk_scsi_persistent_reserve_out(ISTGT_LU_DISK *spec,
 			xfree(spec_bkp->rsv_port);
 		xfree(spec_bkp);
 	}
+	if (ret == -1 && initiator_ports) {
+		for (i = 0; i < maxports; i++) {
```

Comment:
lets initialize maxports as 0

Reply:
lets initialize initiator_ports to NULL when it got assigned to other structure variable

Reply:
ok